pkgs/top-level/aliases.nix

```diff
@@ -70,6 +70,7 @@ mapAliases ({
   casperjs = throw "casperjs has been removed, it was abandoned by upstream and broken.";
   catfish = xfce.catfish; # added 2019-12-22
   cgmanager = throw "cgmanager was deprecated by lxc and therefore removed from nixpkgs."; # added 2020-06-05
+  chatty-with-plugins = chatty; # added 2020-08-16
```

Comment:
Why add an legacy alias when this never existed?

Reply:
Please remove this. Aliases are for package renames/moves/etc.